docs/content/components/splitter.md

```diff
@@ -0,0 +1,284 @@
+
+<Badge>Alpha</Badge>
+
+<Description>
+A component that divides your layout into resizable sections.
+</Description>
+
+<ComponentPreview name="Splitter" />
+
+::: info Credit
+This component was heavily inspired by [react-resizable-panels](https://github.com/bvaughn/react-resizable-panels) by [Bryan Vaughn](https://github.com/bvaughn).
+:::
+
+## Features
+
+<Highlights
+  :features="[
+    'Supports keyboard interaction.',
+    'Supports horizontal/vertical layout.',
+    'Supports nested layout.',
+    'Supports Right to Left direction.',
+    'Can resize across another panel.',
+    'Can be mounted conditionally.'
+  ]"
+/>
+ 
+## Installation
+
+Install the component from your command line.
+
+```bash
+npm install radix-vue
+```
+
+## Anatomy
+
+Import all parts and piece them together.
+
+```vue
+<script setup>
+import { SplitterGroup, SplitterPanel, SplitterResizeHandle } from 'radix-vue'
+</script>
+
+<template>
+  <SplitterGroup>
+    <SplitterPanel />
+    <SplitterResizeHandle />
+  </SplitterGroup>
+</template>
+```
+
+## API Reference
+
+### Group
+
+Contains all the parts of a Splitter.
+
+
+<!-- @include: @/meta/SplitterGroup.md -->
+
+
+<DataAttributesTable
+  :data="[
+    {
+      attribute: '[data-orientation]',
+      values: ['vertical', 'horizontal'],
+    },
+    {
+      attribute: '[data-state]',
+      values: ['collapsed', 'expanded', 'Present when collapsbile'],
+    },
+  ]"
+/>
+
+### Panel
+
+A collapsible section.
+
+<!-- @include: @/meta/SplitterPanel.md -->
+
+ 
+### Resize Handle
+
+Handle that use for resizing.
+
+<!-- @include: @/meta/SplitterResizeHandle.md -->
+
+<DataAttributesTable 
+  :data="[
+    {
+      attribute: '[data-state]',
+      values: ['drag', 'hover', 'inactive'],
+    },
+    {
+      attribute: '[data-disabled]',
+      values: 'Present when disabled',
+    },
+    {
+      attribute: '[data-orientation]',
+      values: ['vertical', 'horizontal'],
+    }
+  ]" 
+/>
+
+
+## Examples
+
+### Collapsible
+
+Use the `collapsible` prop to allow the panel to collapse into `collapsedSize` when `minSize` is reached.
+
+(`collapsedSize` and `minSize` props are required.)
+
+```vue line=2
+<template>
+  <SplitterGroup>
+    <SplitterPanel collapsible :collapsed-size="10" :min-size="35">
+      Panel A
+    </SplitterPanel>
+    <SplitterResizeHandle />
+    <SplitterPanel>
+      Panel B
+    </SplitterPanel>
+  </SplitterGroup>
+</template>
+```
+
+### Persist in localStorage
+
+Use the `autoSaveId` prop to save the layout data into `localStorage`.
+
+```vue line=2
+<template>
+  <SplitterGroup auto-save-id="any-id">
+    …
+  </SplitterGroup>
+</template>
+```
+ 
+### Persist layout with SSR
+
+By default, Splitter uses `localStorage` to persist layouts. With server rendering, this can cause a flicker when the default layout (rendered on the server) is replaced with the persisted layout (in localStorage). The way to avoid this flicker is to also persist the layout with a cookie like so:
```

Comment:
```suggestion
By default, Splitter uses `localStorage` to persist layouts. With server rendering, this can cause a flicker when the default layout (rendered on the server) is replaced with the persisted layout (in `localStorage`). The way to avoid this flicker is to also persist the layout with a cookie like so:
```